var/spack/repos/builtin/packages/h5bench/package.py

```diff
@@ -9,10 +9,17 @@
 class H5bench(CMakePackage):
     """A benchmark suite for measuring HDF5 performance."""
 
-    homepage = "https://github.com/hpc-io/h5bench"
-    git      = "https://github.com/hpc-io/h5bench.git"
+    homepage = 'https://github.com/hpc-io/h5bench'
+    git      = 'https://github.com/hpc-io/h5bench.git'
+    url      = 'https://github.com/hpc-io/h5bench/archive/refs/tags/1.0.tar.gz'
+
+    maintainers = ['jeanbez', 'sbyna']
 
     version('master', branch='master')
+    version('develop', branch='develop')
+
+    version('1.0', sha256='c9151d0c138990f7fc684501f7a7e99d8727317b5169809ddbb63d8e84c9fa3f', deprecated=True)
+    version('1.1', sha256='8d7ba7d835a9a08d88b1a9c6289eafbf67e3a1ea87435799f276809db9df3d77')
```

Comment:
Note: `spack` assumes the default version to be the version called first in the class.

If you didn't actually want to mark `1.0` as `deprecated=True` and only did it to make `1.1` the default here, you can exchange the two lines and remove the ', deprecated = True`.
```suggestion
    version('1.1', sha256='8d7ba7d835a9a08d88b1a9c6289eafbf67e3a1ea87435799f276809db9df3d77')
    version('1.0', sha256='c9151d0c138990f7fc684501f7a7e99d8727317b5169809ddbb63d8e84c9fa3f')
```
For now not it's not bad to deprecate `1.0`, but in case you push an update to this PR (which causes additional delay and CO2 usage as the check and pipelines have to run again then), you can apply this suggestion. This is merely a note for future updates.

Reply:
Thanks for letting me know! Since we needed to fix the sha256 I've updated that as well.